src/sentry/api/endpoints/chunk.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import absolute_import
+
+import six
+import jsonschema
+
+from rest_framework import status
+from rest_framework.response import Response
+from django.core.urlresolvers import reverse
+
+from sentry import options
+from sentry.utils import json
+from sentry.models import File, FileBlob
+from sentry.models.file import DEFAULT_BLOB_SIZE, ChunkFileState, ChunkAssembleType
+from sentry.api.base import Endpoint
+from sentry.api.bases.project import ProjectReleasePermission
+
+
+MAX_CHUNKS_PER_REQUEST = 16
+MAX_CONCURRENCY = 4
+HASH_ALGORITHM = 'sha1'
+
+
+class ChunkUploadEndpoint(Endpoint):
+    permission_classes = (ProjectReleasePermission, )
+
+    def get(self, request):
+        """
+        Return chunk upload parameters
+        ``````````````````````````````
+        :auth: required
+        """
+        endpoint = options.get('system.upload-url-prefix')
+        # We fallback to default system url if config is not set
+        if len(endpoint) == 0:
+            endpoint = options.get('system.url-prefix')
+
+        return Response(
+            {
+                'url': '{}{}'.format(endpoint, reverse('sentry-api-0-chunk-upload')),
```

Comment:
This is the only thing that changed here.
We want to return the full url instead of just the "domain"